src/libstd/process.rs

```diff
@@ -742,21 +742,128 @@ impl fmt::Debug for Output {
     }
 }
 
-/// Describes what to do with a standard I/O stream for a child process.
+/// Describes what to do with a standard I/O stream for a child process when
+/// passed to the [`stdin`], [`stdout`], and [`stderr`] methods of [`Command`].
+///
+/// [`stdin`]: ./struct.Command.html#method.stdin
```

Comment:
I don't know much about rustdoc but these are the only instances of `grep -r '\./struct'` in the whole repo. Could you check whether other docs use a different style for links?

Reply:
Sure, I'll take a look and I'll fix the trailing whitespace.